pyproject.toml

```diff
@@ -87,3 +87,6 @@ write_to = "rsconnect/version.py"
 
 [tool.pytest.ini_options]
 markers = ["vetiver: tests for vetiver"]
+
+[tool.pyright]
+typeCheckingMode = "strict"
```

Comment:
I added this configuration for pyright/pylance, setting it to strict mode. For developers who work in VS Code and use pylance, this will encourage adding more accurate types over time. (I'm happy to undo this if it's too much.)